REVIEW_GUIDELINES.md

```diff
@@ -44,8 +45,29 @@ labels:
   heritage: {{ .Release.Service }}
 ```
 
+If a chart has multiple components, a `component` label should be added (e. g. `component: server`). The resource name should get the component as suffix (e. g. `name: {{ template "myapp.fullname" . }}-server`).
+
 Note that templates have to be namespaced. With Helm 2.7+, `helm create` does this out-of-the-box. The `app` label should use the `name` template, not `fullname` as is still the case with older charts.
 
+In case of a controller like `Deployment` or `StatefulSet`, those labels should also be set in `spec.template.metadata.labels`.
```

Comment:
Is it true even for `heritage`?

Reply:
After further thought, perhaps this paragraph is unnecessary? Now that this PR is separating concerns between `Metadata` and `Deployments and StatefulSets selectors`, I'm not sure the goal of this paragraph.

If the goal is to clarify that by metadata labels we mean `spec.template.metadata.labels`, then perhaps add that to the line above? Something like this:
```diff
-The standard resource metadata should be this:
+The standard resource metadata (`spec.template.metadata.labels`) should be this:
```

Reply:
All right! let me propose then:

```diff
-The standard resource metadata should be this:
+The standard resource metadata (`metadata.labels` and `spec.template.metadata.labels`) should be this:
```

+ removing this paragraph.

Reply:
Done.